devtools/conda-recipe/meta.yaml

```diff
@@ -31,14 +31,12 @@ test:
     - nbformat
     - ipykernel
     - ipyext
+    - python-coveralls
+    - pytest-cov
 
-  source_files:
-    - mbuild/examples/*
-
-  commands:
-    - py.test -v --pyargs mbuild
-    - cd mbuild/examples
-    - py.test test_examples.py -v
+#  commands:
+#    - py.test -v --cov=mbuild --cov-report= --pyargs mbuild
+#    - coveralls 
```

Comment:
I see what you mean now--I need to change the appveyor.yml to mirror this, I didn't touch any appveyor stuff in this PR.

I'll see what I can do to make travis and appveyor as mirrored as possible.